src/Generating/BioGen.cpp

```diff
@@ -796,7 +796,7 @@ EMCSBiome cBioGenTwoLevel::SelectBiome(int a_BiomeGroup, size_t a_BiomeIdx, int
 		EMCSBiome OuterBiome;
 	} ;
 
-	static BiomeLevels bgOcean[] =
+	static BiomeLevels bgOceanBlocks[] =
```

Comment:
Was this a warning??

Reply:
Yes- there is another variable in the global scope with the same name